compute/admin_guide/install/install-gke.adoc

```diff
@@ -33,6 +33,7 @@ You'll also need access to a more privileged GCP account that has the *Kubernete
 * You have deployed a GKE cluster.
 * You have a Google Cloud Platform (GCP) service account with the *Kubernetes Engine Developer* role.
 * You have access to a GCP account with at least the *Kubernetes Engine Admin* role.
+* Prepare your workloads for https://cloud.google.com/kubernetes-engine/docs/how-to/prepare-arm-workloads-for-deployment#node-affinity-multi-arch-arm[multiple architectures in GKE].
```

Comment:
@rcaballeromx we need to put it after "step 2: With the Kubernetes Engine Developer service account, continue with the standard Kubernetes install procedure for Prisma Cloud Console and Defenders, [starting here](https://docs.paloaltonetworks.com/prisma/prisma-cloud/22-06/prisma-cloud-compute-edition-admin/install/install-gke#_gke__install_console)." so that the customer will be able to edit the defender yaml file so it could run on ARM nodes.

I also noticed that the [starting here](https://docs.paloaltonetworks.com/prisma/prisma-cloud/22-06/prisma-cloud-compute-edition-admin/install/install-gke#_gke__install_console) link doesn't redirect to the same page, and not to the Kubernetes installation

Reply:
According to the Google's docs
"If you want to schedule a workload across multiple architectures including Arm, you can specify this in different ways, depending on whether you want the workload to be constrained to only some architectures or be scheduled to any of the available architectures"

Reply:
@MorShabiPAN I looked into how and where to best provide the reference to the GKE documentation. I left one reference here and updated the cross-references on the step you mentioned too. PTAL
I didn't add the pre-requisite to the step on this page. Instead, I added the prerequisite to the Defender installation on GKE page to provide visibility there too.

Reply:
Thanks @rcaballeromx, the only thing I would add is that the customer need to do this when they are using multi-arch nodes as well as when they are using only ARM nodes. So that I don't what the customer to pass through these instructions when they are using ARM only

Reply:
I added some text to call out ARM specifically. @MorShabiPAN